config/env/development.js

```diff
@@ -30,5 +30,5 @@ module.exports = {
     callbackURL: '/api/auth/facebook/callback'
   },
   livereload: true,
-  seedDB: process.env.MONGO_SEED || false
+  seedDB: process.env.MONGO_SEED || true
```

Comment:
Default to true - not sure if there was a reason why we weren't using the seed before @cloudier

Reply:
Think this was part of the boilerplate